bigtable/client/filters.h

```diff
@@ -71,8 +74,12 @@ class Filter {
     return tmp;
   }
 
-  /// Create a filter that accepts only the last @a n values.
-  static Filter Latest(int n) {
+  /**
+   * Return a filter that accepts only the last @p n values of each column.
+   *
+   * TODO(#84) - document what is the effect of n <= 0
+   */
+  static Filter Latest(std::int32_t n) {
```

Comment:
Can we use an unsigned int here, or is int32 the correct type to pass down the call stack?

Reply:
The rest of the stack expects a `int32`.